test/metabase/notification/test_util.clj

```diff
@@ -102,6 +102,18 @@
 (def default-card-name "Card notification test card")
 
 #_{:clj-kondo/ignore [:metabase/test-helpers-use-non-thread-safe-functions]}
+(defn with-temp-notification
```

Comment:
Even though we can't make this private, as its used directly by a macro, perhaps we could still indicate that its an internal implementation detail but putting a star at the end of the name?

Reply:
Actually, I think we can just make it private?